src/ducks/categories/Categories.jsx

```diff
@@ -218,19 +243,30 @@ class Categories extends Component {
         className={isCollapsed ? stRow : stUncollapsed}
         onClick={() => this.handleClick(category, subcategory)}
       >
-        <td className="u-ph-1 u-pv-half">
+        <td className="u-ph-1">
           <Media>
             <Img className="u-pr-half" style={subcategory ? vHidden : null}>
               <CategoryIcon category={categoryName} style={!subcategory} />
             </Img>
             <Bd className={cx('u-ph-half', stCategory)}>
-              {t(`Data.${type}.${name}`)}
+              <ListItemText>
+                <Text>{t(`Data.${type}.${name}`)}</Text>
+                <Caption className={styles['bnk-table-row-caption']}>
+                  <span className={styles['bnk-table-percentage-caption']}>
+                    {!subcategory && selectedCategoryName
+                      ? '100%'
+                      : `${percentage}%`}
+                  </span>
+                  <span className={styles['bnk-table-operation-caption']}>
+                    {`${transactionsNumber} ${t(
+                      `Categories.headers.transactions.${
+                        transactionsNumber > 1 ? 'single' : 'plural'
```

Comment:
FYI http://airbnb.io/polyglot.js/ has a smart_count feature. Not blocking at all.